src/internal/operators/every.ts

```diff
@@ -7,10 +7,11 @@ export function every<T>(
   predicate: BooleanConstructor,
   thisArg?: any
 ): OperatorFunction<T, Exclude<T, Falsy> extends never ? false : boolean>;
-export function every<T>(
-  predicate: (value: T, index: number, source: Observable<T>) => boolean,
-  thisArg?: any
+export function every<T, A>(
+  predicate: (this: A, value: T, index: number, source: Observable<T>) => boolean,
+  thisArg: A
```

Comment:
As an aside, I'd like to deprecate everywhere we're doing `thisArg`. Because it's a goofy pattern, and `.bind()` is more obvious: `every(fn, obj)` vs `every(fn.bind(obj))`. (Also closures are probably more readable given how most people would use this, if it's even used)